mysql-test/suite/innodb/t/insert_into_empty.test

```diff
@@ -495,3 +495,16 @@ DROP TABLE t1;
 CREATE TABLE t (a CHAR CHARACTER SET utf8) ENGINE=InnoDB ROW_FORMAT=REDUNDANT;
 INSERT t SELECT left(seq,1) FROM seq_1_to_43691;
 DROP TABLE t;
+
+--echo #
+--echo #  MDEV-32453  Bulk insert fails to apply when trigger
+--echo #                  does insert operation
+--echo #
+SET AUTOCOMMIT=0;
+CREATE TABLE t(c INT)ENGINE=InnoDB;
+CREATE TRIGGER t2_ai AFTER INSERT ON t FOR EACH ROW SET @a:=(SELECT * FROM t);
+INSERT INTO t VALUES (1);
+--error ER_SUBQUERY_NO_1_ROW
+INSERT INTO t VALUES (1);
```

Comment:
Instead of changing a session variable, I’d use ordinary `BEGIN` and `COMMIT` around the two `INSERT` statements if this bug is repeatable with those as well. Alternatively, maybe this would be reproducible with a two-row `INSERT` in autocommit mode:
```sql
--error ER_SUBQUERY_NO_1_ROW
INSERT INTO t VALUES(1),(1);
```
I would also add `--echo # End of 10.11 tests` to the end of this file.